docs/mobile_data_collected.md

```diff
@@ -0,0 +1,173 @@
+#Mobile Data Collected
+
+The Datadog Real User Monitoring SDK generates six types of events:
+
+| Event Type | Description                                                                                                                                                                                                                                                   |
```

Comment:
This table seems broken when I [preview its GH rendered page](https://github.com/DataDog/dd-sdk-ios/blob/priyanshi/data_collected/docs/mobile_data_collected.md).